opentelemetry-sdk/src/trace/config.rs

```diff
@@ -43,6 +43,19 @@ impl Default for Config {
             config.span_limits.max_attributes_per_span = max_attributes_per_span;
         }
 
+        if let Some(max_attribute_value_length) = env::var("OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT")
+            .ok()
+            .and_then(|count_limit| i32::from_str(&count_limit).ok())
+        {
+            config.span_limits.max_attribute_value_length = max_attribute_value_length;
+        } else if let Some(max_attribute_value_length) =
+            env::var("OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT")
```

Comment:
supporting OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT looks reasonable, but OTEL_ATTRIBUTE_VALUE_LENGTH_LIMIT is not something we can add right now, as Logs don't respect this at all.